src/Components/getList.js

```diff
@@ -27,16 +27,67 @@ function EmptyList() {
   );
 }
 
-function FullList(props) {
+
+const FullList = (props)=> {
   const [isChecked, setCheck] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
```

Comment:
I like how you gave this a default value of `""`. I tend to keep things non-nullable if possible! 👍